src/Bicep.LangServer/BicepParamsCompilationManager.cs

```diff
@@ -49,5 +56,14 @@ public void CloseCompilation(DocumentUri uri)
             return context;
         }
 
+        private void PublishEmptyDocumentDiagnostics(DocumentUri uri, int? version)
+        {
+            server.TextDocument.PublishDiagnostics(new PublishDiagnosticsParams
+            {
+                Uri = uri,
+                Version = version,
+                Diagnostics = new Container<Diagnostic>()
+            });
+        }
```

Comment:
I don't think you need a `PublishEmpty...` method here. The real version in BicepCompilationManager is:

```csharp
private void PublishDocumentDiagnostics(DocumentUri uri, int? version, IEnumerable<Diagnostic> diagnostics)
{
    server.TextDocument.PublishDiagnostics(new PublishDiagnosticsParams
    {
        Uri = uri,
        Version = version,
        Diagnostics = new Container<Diagnostic>(diagnostics)
    });
}
```

And you can just pass an empty enumerable on line 45

Reply:
You mean I should reuse the method? Me and Anthony decided to put the empty method there temporarily to unblock progress on integration test for semantic token handler

Reply:
It's not important, since both approaches have the same end effect (publishing 0 diagnostics for the document at the provided Uri). I was just confused about the intent here since the prior art implementation would accept an empty collection of diagnostics.

Reply:
Either works! I suggested `PublishEmptyDocumentDiagnostics` just to draw attention to the fact that it's a little quirky - essentially we're only doing this to make the test framework happy.